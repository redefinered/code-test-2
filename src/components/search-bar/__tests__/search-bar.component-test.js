import React from 'react';
import SearchBar from '../search-bar.component';

import renderer from 'react-test-renderer';

describe('search-bar tests', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<SearchBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
