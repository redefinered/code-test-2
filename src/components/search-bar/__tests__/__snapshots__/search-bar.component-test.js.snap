// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search-bar tests renders correctly 1`] = `
<TextInput
  autoFocus={true}
  clearButtonMode="always"
  onChangeText={[Function]}
  returnKeyType="search"
  style={
    Object {
      "borderRadius": 10,
      "borderWidth": 1,
      "marginBottom": 10,
      "padding": 10,
    }
  }
  value={Array []}
/>
`;
